degov_paragraph_slideshow/js/slideshow.js

```diff
@@ -63,23 +63,23 @@
       function get_prev_slick_img() {
         var $slick_current = $('.slick-current');
         // For prev img
-        var prev_slick_img = $slick_current.prev('.slide').find('.slide__media img').attr('src');
+        var prev_slick_img = $slick_current.prev('.slick-slide').find('.slide__media img').attr('src');
         $('.prev-slick-img img').attr('src', prev_slick_img);
         $('.prev-slick-img').css('background-image', 'url(' + prev_slick_img + ')');
         // For next img
-        var prev_next_slick_img = $slick_current.next('.slide').find('.slide__media img').attr('src');
+        var prev_next_slick_img = $slick_current.next('.slick-slide').find('.slide__media img').attr('src');
```

Comment:
If you reference the node in the slide it doesn't have class slide, but all slides have class slick-slide.